examples/test_all/fibonacci/README.md

```diff
@@ -11,9 +11,7 @@ cmake --build .
 
 ## Running the sample
 ### Predefined make targets
-* `make run_fractal` - executes the example with predefined parameters.
-* `make perf_run_fractal` - executes the example with suggested parameters to measure the oneTBB performance.
-* `make light_test_fractal` - executes the example with suggested parameters to reduce execution time.
+* `make run_fibonacci` - executes the example with predefined parameters.
```

Comment:
Shouldn't we mention make perf_run_fibonacci as well?

Reply:
Nevermind! I checked the cmakeLists.txt of Fibonacci doesn't have that target :)